PowerEditor/installer/nativeLang/english.xml

```diff
@@ -1528,7 +1531,7 @@ Do you want to save your changes before switching themes?"/> <!-- HowToReproduce
             </Menus>
         </ProjectManager>
         <MiscStrings>
-            <!-- $INT_REPLACE$  and $STR_REPLACE$ are a place holders, don't translate these place holders -->
+            <!-- $INT_REPLACE$ and $STR_REPLACE$ are a place holders, don't translate these place holders -->
```

Comment:
Also just comments:
$INT_REPLACE$ and $STR_REPLACE$ are a place holders -> $INT_REPLACE$ and $STR_REPLACE$ are place holders